oci_script

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/python
+from oci import main
+main.run()
```

Comment:
Please add a new line to the end of the file.

The file should be named "oci", and be executable.

You should be able to run it from source like this:

    ./oci build-artifacts 12345

Reply:
Lets rename the script to ovirt-ci:

    ovirt-ci build-artifacts 12345

See also #31

Reply:
Done